### ISSUE TITLE
path to changelog.gif missing

### ISSUE THREAD

**Issue by fritzmg** [contao/core#8285]
In `3.5.9` [this line](https://github.com/contao/core/blob/3.5.9/system/modules/core/classes/LiveUpdate.php#L46) has been changed from
```php
…<img src="' . TL_FILES_URL . 'system/themes/' . \Backend::getTheme() . '/images/changelog.gif" width="14" height="14" alt="" style="vertical-align:text-bottom;padding-left:3px">…
```
to
```php
…' . \Image::get('changelog.gif', '', 'style="vertical-align:text-bottom;padding-left:3px"') . '…
```
`\Image::get` does not automatically add the path to the image folder of the current backend theme, or does it? [One user reports](https://community.contao.org/de/showthread.php?61834-Contao-3-5-9-verf%FCgbar&p=403573&viewfull=1#post403573)
```
Image "changelog.gif" could not be processed: Image "changelog.gif" could not be found 
```
entries in his system log after updating to `3.5.9`.

**Comment by BugBuster1701** [comment]
Sollte da eventuell `\Image::getHtml` statt `\Image::get` stehen?

**Comment by fritzmg** [comment]
Indeed. But the full path to the file is still required.

**Comment by leofeyer** [comment]
No it is not if you use `Image::getHtml()` or `Image::getPath()`.

**Comment by fritzmg** [comment]
Ah, right. Missed to call to `Image::getPath()` in `Image::getHtml()`.

**Comment by leofeyer** [comment]
Fixed in cbf025662c6b5e761e11353f7892efa0706acc72.
